drivers/net/ethernet/renesas/rswitch.c

```diff
@@ -4299,6 +4299,54 @@ static void rswitch_fwd_init(struct rswitch_private *priv)
 		   priv->addr + FWCMPTC);
 }
 
+static void rswitch_set_max_hash_collisions(struct rswitch_private *priv)
+{
+	/* 320Mhz */
+	const u64 clk_freq = 320 * 1000 * 1000;
+	/* Avarage frame size in bits */
+	const u64 average_frame_size = 64 * 8;
```

Comment:
It is customary to hold such constans at the beginning of the file, as `#define`

Reply:
Good point, changed to define.